erpnext/projects/doctype/project/project.json

```diff
@@ -121,6 +127,35 @@
    "allow_on_submit": 0, 
    "bold": 0, 
    "collapsible": 0, 
+   "columns": 0, 
+   "default": "Task Completion", 
+   "fieldname": "pct_comp_method", 
```

Comment:
Always use full names `percent_complete_method`

Also maybe this should be in the "Tasks" section rather than the header section